packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormAmountInput/TransferFormAmountInput.tsx

```diff
@@ -25,20 +25,21 @@ const TransferFormAmountInput: React.FC<TProps> = ({ fieldName }) => {
         return newRates;
     }, [refetchAccountLimits, refetchExchangeRates]);
 
-    const isFromAmountFieldName = fieldName === 'fromAmount';
+    const hasNoMoney = Number(fromAccount?.balance) === 0;
```

Comment:
hey @rostislav-deriv 
my first concern is:
maybe it's better to create hasMoney variable than adding `No` to the name
my second concern is:
how about renaming it to hasFunds = Number(fromAccount?.balance) > 0;
?

Reply:
Indeed, done